shephard-core/src/main/java/com/devsda/platform/shepherdcore/service/documentservice/ExecutionDocumentConstants.java

```diff
@@ -8,6 +8,10 @@
         public static final String OBJECT_ID = "objectId";
         public static final String LAST_MODIFIED_DATE = "lastModifiedDate";
         public static final String UPDATE_COUNT = "numberOfUpdatesAfterInsertion";
+        public static final String CLIENT_ID = "clientId";
```

Comment:
All constants should be a part of Shepherd-core level constants file. Please check. 

You can create one class inside same Constants file. i.e, public static final class ExecutionDocument {....}

Reply:
Remove this file.